src/scp/SCP.h

```diff
@@ -24,6 +26,8 @@ typedef std::shared_ptr<SCPQuorumSet> SCPQuorumSetPtr;
 class SCP
 {
     SCPDriver& mDriver;
+    std::unordered_map<NodeID, std::shared_ptr<medida::Histogram>>
```

Comment:
I found it convenient to use histograms in order to track nodes voting overtime. However, right now the sliding window defaults to 5 min of data, and I'm wondering if this is too short (we could change medida to allow passing in a longer time window)